py/desiutil/dust.py

```diff
@@ -482,3 +670,78 @@ def ebv(*args, **kwargs):
                south=kwargs.get('south', "SFD_dust_4096_sgp.fits"),
                scaling=kwargs.get('scaling', 1.))
     return m.ebv(*args, **kwargs)
+
+def main():
```

Comment:
Is this `main` leftover from debugging, or is it still useful?  I'm ok with sometimes hiding super-specialized scripts under `if __name__ == "__main__":` blocks under otherwise module code, but if you do want to retain this please add some brief comments about what it does and add a basic `argparse` so that `python dust.py --help` would be informative without actually running anything.